src/python/turicreate/toolkits/drawing_classifier/_tf_drawing_classifier.py

```diff
@@ -182,7 +182,8 @@ def predict(self, feed_dict):
         if is_train:
             # convert to one hot
             labels = feed_dict["labels"].astype("int32").T
-            one_hot_labels[_np.arange(num_samples), labels] = 1
+            one_hot_labels[_np.arange(int(num_samples)), labels] = 1
```

Comment:
Why is num_samples not an int in the first place?

Reply:
Well when you send it from C++, we send it as `shared_float_array` which is float32.

Reply:
Can you add a simple unit test to make sure we don't crash again.

Reply:
Added.